enterprise/management/commands/email_drip_for_missing_dsc_records.py

```diff
@@ -88,7 +89,7 @@ def get_enterprise_course_enrollments(self, options):
                 enterprise_customer_user__enterprise_customer__enable_data_sharing_consent=True
             )
         else:
-            past_date = date.today() - timedelta(days=past_num_days)
+            past_date = timezone.now().date() - timedelta(days=past_num_days)
```

Comment:
I would recommend to not change the management command. Instead update the test only.